control-plane-operator/controllers/hostedcontrolplane/pki/ignitionserver.go

```diff
@@ -0,0 +1,18 @@
+package pki
+
+import (
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/openshift/hypershift/support/config"
+)
+
+func ReconcileIgnitionServerCertSecret(secret, ca *corev1.Secret, ownerRef config.OwnerRef) error {
```

Comment:
We need to find a way to allow opting out of the proxy, otherwise this would require a new cert that would break IBM on a z release.

Reply:
Do we have a preferred method?